BoxContentSDK/BoxContentSDK/Requests/BOXRequest.h

```diff
@@ -70,6 +70,15 @@ typedef void (^BOXMetadataTemplatesBlock) (NSArray <BOXMetadataTemplate *>*metad
 
 typedef void (^BOXRepresentationInfoBlock)(BOXRepresentation *representation, NSError *error);
 
+@protocol BOXBackgroundRequestProtocol <NSObject>
```

Comment:
Minor: I don't think this needs to conform to `NSObject`

Reply:
done